kong.conf.default

```diff
@@ -1826,3 +1826,29 @@ G# -----------------------
 
 #wasm_filters_path =              # Path to the directory containing Wasm filters
                                   # that Kong must load on startup.
+
+#wasm_shm_kv =                    # Define a shared key/value memory zone accessible
+                                  # through all Contexts and by all nginx worker
```

Comment:
What are "all Contexts"? Is that qualifier necessary, if so why?

Reply:
Contexts are the directives like `http {}`, `stream {}`, `server {}` and so on in `nginx.conf`. That doesn't matter from a Kong user's perspective, I removed it.